.circleci/config.yml

```diff
@@ -81,6 +81,13 @@ jobs:
               echo Failing build due to less than 80% test coverage
               exit 1
             fi
+      - run:
+          name: Report failing Master
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
we should make this an executable script in `./scripts/notify_failure.sh` to simplify adding it to more job nodes.